ansible/slave.yml

```diff
@@ -81,6 +81,13 @@
       # docs.ceph.com.pub is the output of `ssh-keyscan docs.ceph.com`
       key="{{ lookup('file', 'ssh/hostkeys/docs.ceph.com.pub') }}"
 
+    - name: add github.com host key
+      sudo: true
+      known_hosts: path='/etc/ssh/ssh_known_hosts'
```

Comment:
I don't think that key=value works here with newlines. I think you'll want key: value, each on a new line.